### PR TITLE
Fix of NPE in TruffleAccess.

### PR DESCRIPTION
**Pull request by entlicher**
Fix of a NPE reported internally:
```
java.lang.NullPointerExceptionjava.lang.NullPointerException
    at org.netbeans.modules.debugger.jpda.truffle.access.TruffleAccess.getBreakpointInfos(TruffleAccess.java:460)
```

### CODE REVIEW

**Review by neilcsmith-net — APPROVED** [review]
OK, I'll merge once @JaroslavTulach has reviewed. RC3 will be a few days late waiting on Gradle full release anyway.

File: java/debugger.jpda.truffle/src/org/netbeans/modules/debugger/jpda/truffle/access/TruffleAccess.java
Comment (JaroslavTulach) [inline java/debugger.jpda.truffle/src/org/netbeans/modules/debugger/jpda/truffle/access/TruffleAccess.java:460]: No need to put `(...)` around the condition. All condition operators have higher precedence than `?:`.

**Review by JaroslavTulach — APPROVED** [review]

File: java/debugger.jpda.truffle/src/org/netbeans/modules/debugger/jpda/truffle/access/TruffleAccess.java
Comment (entlicher) [reply java/debugger.jpda.truffle/src/org/netbeans/modules/debugger/jpda/truffle/access/TruffleAccess.java:460]: I know. I'm just used to it to visually enclose the condition for better readability.

**Pull request merged**
